README.md

```diff
@@ -83,14 +22,25 @@ You can override the default one, see all configuration option on
 
 ### Trust store [src/Service/InMemoryAttestationCertificateTrustStore.php](src/Service/InMemoryAttestationCertificateTrustStore.php)
 
-All WebAuthn request Attestation Statement with a certificate trust path. All trusted certificates should be stored on disk.
+- Only allows WebAuthn Attestation Statements with a certificate trust path. 
```

Comment:
This sentence is rather confusing. What are you explaining here?

Reply:
There are different Attestation Statements without certificate trust paths, those are not supported.